balance_check/validators/gift_card.py

```diff
@@ -0,0 +1,44 @@
+import re
+from enum import Enum
+from luhn import verify
+
+
+class Merchant(Enum):
+    GameStop = 'GameStop'
+
+
+merchant_regex = {
+    Merchant.GameStop: re.compile('636491[0-9]{13}')
+}
+
+
+def luhn_check(field, value, error):
+    if not verify(value):
+        error(field, "does not pass Luhn check")
+
+
+def GiftCardSchema(merchants):
```

Comment:
This should only ever be a validating a single merchant, right? This doesn't need to be a list if that's the case.

```suggestion
def GiftCardSchema(merchant):
```